bdns/dns.go

```diff
@@ -114,6 +115,73 @@ var (
 			Mask: []byte{255, 192, 0, 0},
 		},
 	}
+	privateV6Networks = []net.IPNet{
```

Comment:
Did you decide against incorporating Roland's iana_spar package, or just decide to leave that for a later change?